### PR TITLE
Fill solana network ids with correct value

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #1244

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@aminlatifi  Now I fill the `networkId` in backend forthese web services:
* createDonation
* updateProject
* createProject
* updateProjectVerificationForm

Also we had two migration files for solana I checked the production and saw just one of them has been executed so I edited the second one for adding spl tokens in db

File: migration/data/seedTokens.ts [hunk 1]
```diff
@@ -1344,6 +1344,38 @@ const seedTokens: ITokenData[] = [
...
+    chainType: ChainType.SOLANA,
+  },
+
+  // SOLANA testnet
+  {
+    name: 'Solana native token',
+    symbol: 'SOL',
+    address: SOLANA_SYSTEM_PROGRAM,
+    decimals: 9,
+    networkId: NETWORK_IDS.SOLANA_DEVNET,
```
Comment (aminlatifi) [inline migration/data/seedTokens.ts]:
```suggestion
    networkId: NETWORK_IDS.SOLANA_TESTNET,
```

File: src/repositories/projectRepository.ts [hunk 2]
```diff
@@ -296,7 +298,13 @@ export const updateProjectWithVerificationForm = async (
     await ProjectAddress.create({
       title: relatedAddress.title,
       address: relatedAddress.address,
-      networkId: relatedAddress.networkId,
+
+      // Frontend doesn't send networkId for solana addresses so we set it to default solana chain id
+      networkId:
+        relatedAddress.chainType === ChainType.SOLANA
+          ? getDefaultSolanaChainId()
+          : relatedAddress.networkId,
```
Comment (aminlatifi) [inline src/repositories/projectRepository.ts]: If it was me, I would move this logic to a method as it is repeated plenty of times.

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Review by aminlatifi — APPROVED** [review]

File: src/repositories/projectRepository.ts (hunk 2 above)
Comment (mohammadranjbarz) [reply src/repositories/projectRepository.ts]: You are right, done.

File: migration/data/seedTokens.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply migration/data/seedTokens.ts]: Done

**Comment by aminlatifi** [comment]
@mohammadranjbarz Please be sure the vault is filled with appropriate values (staging + production) and merge.

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Please be sure the vault is filled with appropriate values (staging + production) and merge.

@aminlatifi Thanks for reminding, done

**Pull request merged**